client/src/models/covid/populate.js

```diff
@@ -23,7 +23,7 @@ export const api_load_years_with_covid_data = (subject) => {
         return {
           is_loaded: subject_is_loaded(subject.id),
           level: "dept",
-          id: subject.id,
+          id: String(subject.id),
```

Comment:
Apollo server doesn't cast integers to strings the way express-graphql does, so I did that manually. 

We could alternatively make the department queries in the schema take integer arguments 🤷

Reply:
Ouch. Yeah, the inconsistency in id types between int and string bugs me (and creates bugs) a lot. Org ids should probably just always be strings, doesn't really matter that they happen to only have int values. Other subjects pretty much all have non-int ids so it's only the org subjects that have the ambiguity headache.